vendor/engines/split_accounts/app/decorators/split_accounts/split_order_detail_decorator.rb

```diff
@@ -14,6 +14,14 @@ def quantity=(value)
       @quantity_override = BigDecimal(value.to_s)
     end
 
+    def time_data
+      @time_data || super
+    end
+
+    def time_data=(time_data)
```

Comment:
Yeah, I'll change it. It always feels a little asymmetrical to implement one as a method and one as an attr_writer, but it's not a big deal to me.